### DISCUSSION TITLE
Show related plugin model in a netbox core model

### DISCUSSION THREAD

**Discussion by bkampsnl** [netbox-community/netbox#13297] [General]
I am developing a new plugin with some models. Those models are related to netbox core models (for example the device).

Is there a way to show the related plugin models in the device detail page (without adjusting the code of the view of the device detail page) ?

**Comment by candlerb** [comment]
Your plugin can add its own tab to the device detail page:
https://docs.netbox.dev/en/stable/plugins/development/views/#extending-core-views

**Reply by bkampsnl** [reply]
You are absolutely right ! remember reading something that it should be possible but I thought it was on a forum but instead it's a RTFM..

**Comment by vilnitsky** [comment]
Hi! Was looking for the same, was also wondering if it's possible to include related plugin data into the device API?
